crypto/codec/amino.go

```diff
@@ -26,20 +27,25 @@ func RegisterCrypto(cdc *codec.LegacyAmino) {
 	// first line.
 	cdc.RegisterInterface((*crypto.PubKey)(nil), nil)
 	cdc.RegisterInterface((*cryptotypes.PubKey)(nil), nil)
-	cdc.RegisterConcrete(ed25519.PubKey{},
-		ed25519.PubKeyName, nil)
 	cdc.RegisterConcrete(sr25519.PubKey{},
 		sr25519.PubKeyName, nil)
+	// TODO Same as above, for ED25519
+	cdc.RegisterConcrete(tmed25519.PubKey{},
+		tmed25519.PubKeyName, nil)
+	cdc.RegisterConcrete(&ed25519.PubKey{},
+		ed25519.PubKeyName, nil)
 	cdc.RegisterConcrete(&secp256k1.PubKey{},
 		secp256k1.PubKeyName, nil)
 	cdc.RegisterConcrete(&kmultisig.LegacyAminoPubKey{},
 		kmultisig.PubKeyAminoRoute, nil)
 
 	cdc.RegisterInterface((*crypto.PrivKey)(nil), nil)
-	cdc.RegisterConcrete(ed25519.PrivKey{},
-		ed25519.PrivKeyName, nil)
 	cdc.RegisterConcrete(sr25519.PrivKey{},
 		sr25519.PrivKeyName, nil)
+	cdc.RegisterConcrete(tmed25519.PrivKey{},
+		tmed25519.PrivKeyName, nil)
```

Comment:
I guess this we will need to remove as well.

Reply:
added comment.